napari/layers/labels/_tests/test_labels.py

```diff
@@ -293,12 +293,16 @@ def test_custom_color_dict():
     """Test custom color dict."""
     np.random.seed(0)
     data = np.random.randint(20, size=(10, 15))
-    layer = Labels(data, color={1: 'white'})
+    layer = Labels(
```

Comment:
I feel like for a really good test we need a much bigger dataset, and non-consecutive labels should map to the same color. ie

```
{1: 'white', 2: 'red', ..., 18: 'red', 19: 'white'}
```

You'd probably want to construct this programmatically! :grimacing: Although it seems we're ok with missing entries so I guess it doesn't matter.

Reply:
switched to a binary numbering system :)